lib/template/pages/schedule.jade

```diff
@@ -15,7 +15,7 @@
         =' '
         | Lee White Show "Your 15 Minutes of Fame",
         =' '
-        a(href=$ren->url('artists').'#theCrumbs') The Crumbs, CA
+        a(href=$ren->url('artists').'#theCrumbs') The Crumbs, CAN
```

Comment:
CA? CAN? To bude Canada? Canary islands? Obsahově by bylo lepší nepoužívat zkratky.